ofborg is failing with the following:

```c
./screen.c:42:10: fatal error: term.h: No such file or directory
   42 | #include <term.h>
      |          ^~~~~~~~
compilation terminated.
make[2]: *** [Makefile:57: screen.o] Error 1
make[2]: Leaving directory '/build/brltty-6.7/Drivers/Screen/FileViewer'
make[1]: *** [Makefile:1092: screen-drivers] Error 1
make[1]: Leaving directory '/build/brltty-6.7/Programs'
make: *** [Makefile:49: all] Error 2
error: builder for '/nix/store/qwnkxhp80k49gvvra36vjgxf5a1h6lbm-brltty-6.7.drv' failed with exit code 2;
```

--- a/pkgs/by-name/br/brltty/package.nix
+++ b/pkgs/by-name/br/brltty/package.nix
@@ -15,15 +15,16 @@
   alsa-lib,
   systemdSupport ? lib.meta.availableOn stdenv.hostPlatform systemd,
   systemd,
+  ncurses,
 }:
 
 stdenv.mkDerivation rec {
   pname = "brltty";
-  version = "6.6";
+  version = "6.7";
 
   src = fetchurl {
     url = "https://brltty.app/archive/brltty-${version}.tar.gz";
-    sha256 = "E+j2mb8UTuGx6PkAOt03hQkvf1XvEHxJEuPBT2zMpPw=";
+    sha256 = "sha256-FsM9AeL1lnBziJlmB7EZAIgDKylT8D4Il81Fe1y9Yjg=";
   };
 
   nativeBuildInputs = [
@@ -32,7 +33,13 @@ stdenv.mkDerivation rec {
     python3.pkgs.setuptools
     tcl
   ];
-  buildInputs = [ bluez ] ++ lib.optional alsaSupport alsa-lib ++ lib.optional systemdSupport systemd;
+  buildInputs =
+    [
+      bluez
+      ncurses.dev
+    ]
+    ++ lib.optional alsaSupport alsa-lib
+    ++ lib.optional systemdSupport systemd;
 
   meta = {
     description = "Access software for a blind person using a braille display";

